libraries/AP_Logger/AP_Logger.cpp

```diff
@@ -36,7 +36,11 @@ extern const AP_HAL::HAL& hal;
 #endif
 
 #ifndef HAL_LOGGING_STACK_SIZE
-#define HAL_LOGGING_STACK_SIZE 1580
+    #if CONFIG_HAL_BOARD == HAL_BOARD_ESP32
```

Comment:
should be in ESP32 board.h

Reply:
I was unaware that chibios had an internal "extra bit" that it adds to each thread created through Scheduler::thread_create ... i've done equivalent in the esp32 HAL now.